### ISSUE TITLE
Expression type is null when rendering embedded code

### ISSUE THREAD

**Issue by JanMosigItemis** [mbeddr/mbeddr.core#1349]
When trying to embed code into a documention node (D), pointer access is rendered with a dot rather than an arrow. See the following image for details:

![image](https://cloud.githubusercontent.com/assets/16935994/14459379/d6bc55ae-00b7-11e6-93ed-d6e0e78c8279.png)

The reason for this is that function `getRepresentation` in `com.mbeddr.core.expressions.behavior.GenericDotExpression_Behavior` cannot evaluate the type of `this.expression`.

In other words `this.expression.type==null` when the code is rendered into documentation. However it has the correct type when rendered into an implementation module.

This problem needs to be investigated. It may be a bug of MPS itself.

**Comment by alexanderpann** [comment]
Not reproducible anymore.
